xtest/tdfs.py

```diff
@@ -445,6 +448,10 @@ def _uncached_supports(self, feature: feature_type) -> bool:
                 return True
             case ("ns_grants", ("go" | "java")):
                 return True
+            case ("assertion_schema_v2", "go"):
+                # Go SDK supports V2 assertion schema (urn:opentdf:system:metadata:v2)
```

Comment:
Do all supported versions support the v2 schema? You may want to add a feature detector to the go/cli.sh for this somehow, e.g. by adding a note to the appropriate otdfctl --help document.